src/pages/editor/InstitutionSupport.tsx

```diff
@@ -0,0 +1,82 @@
+import { setNotification } from '../../redux/notificationSlice';
+import { RootState } from '../../redux/store';
+import { CustomerInstitution } from '../../types/customerInstitution.types';
+import { dataTestId } from '../../utils/dataTestIds';
+
+export const InstitutionSupport = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const customer = useSelector((store: RootState) => store.customer);
+
+  const customerMutation = useMutation({
+    mutationFn: (values: CustomerInstitution) =>
+      updateCustomerInstitution({ ...values, serviceCenterUri: values.serviceCenterUri }),
+    onSuccess: (response) => {
+      dispatch(setCustomer(response.data));
+      dispatch(setNotification({ message: t('feedback.success.update_customer'), variant: 'success' }));
+    },
+    onError: () => dispatch(setNotification({ message: t('feedback.error.update_customer'), variant: 'error' })),
+  });
+
+  return (
+    <>
+      {!customer ? (
+        <PageSpinner />
+      ) : (
+        <Formik initialValues={customer} onSubmit={async (values) => customerMutation.mutate(values)}>
+          {({ values, isSubmitting, setFieldValue }: FormikProps<CustomerInstitution>) => (
+            <Form style={{ display: 'flex', flexDirection: 'column', height: '100%' }}>
+              <Typography variant="h2">{t('editor.institution.institution_support')}</Typography>
+              <Typography sx={{ fontStyle: 'italic', marginTop: '0.5rem', marginBottom: '1rem' }}>
+                {t('editor.retention_strategy.rrs_required_link')}
+              </Typography>
+              <Typography>{t('editor.institution.institution_support_description')}</Typography>
+
+              <Field name={'serviceCenterUri'}>
+                {({ field }: FieldProps<string>) => (
+                  <TextField
+                    data-testid={dataTestId.editor.institutionSupportInputField}
+                    sx={{ mb: { xs: '1rem', md: 0 }, maxWidth: '40rem' }}
+                    {...field}
+                    disabled={isSubmitting}
+                    value={values.serviceCenterUri}
+                    type="url"
+                    label={t('common.url')}
+                    placeholder={t('editor.retention_strategy.rrs_link')}
+                    variant="filled"
+                    InputProps={{
+                      startAdornment: (
+                        <InputAdornment position="start">
+                          <LinkIcon />
+                        </InputAdornment>
+                      ),
+                    }}
+                    onChange={(event) => {
+                      setFieldValue(field.name, event.target.value);
+                    }}
+                  />
+                )}
+              </Field>
+
+              <Box sx={{ mt: 'auto', display: 'flex', justifyContent: 'center' }}>
+                <LoadingButton variant="contained" type="submit" loading={isSubmitting}>
```

Comment:
trenger ikke egen wrapper-tag her
```suggestion
              <LoadingButton variant="contained" sx={{ alignSelf: 'center', mt: 'auto' }} type="submit" loading={isSubmitting}>
```